server/index.js

```diff
@@ -32,30 +31,27 @@ app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
 
-const isIndentified = (req, res, next) => {
```

Comment:
Why remove this? Identifying the users like this is a good way to make it harder to DoS it with just the 'official client' (web page), even though it won't make much difference otherwise.

Reply:
I have changed the way that user is identified, I suggest using the user IP instead of generated UUID

```js
const token = headers['x-real-ip']
```